lib/lita/handlers/stacker.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
+require 'active_support/core_ext/array/conversions' # Array.to_sentence, for i18n
+
 module Lita
   module Handlers
     class Stacker < Handler
-      VERSION = '0.2.0'
+      VERSION = '1.0.0'
```

Comment:
the interface is backward-compatible, no?  This could also be considered a minor version change due to that, I would think

Reply:
I almost added a comment on this line, but realized it would live past this moment-in-time change so I added it to the git comment (but not my reasoning):

It depends what you consider “the API,” since that’s what SemVer is concerned with. Yes, the user interface underwent a minor version update, but the underlying data interchange is not in any way backward compatible. …But I think I just talked myself out of it, because “your data{model,store,base} is not an API.”

Reply:
Good point about the underlying data being lost with this upgrade.  You've convinced me that it could be considered a major change due to that.